examples/device-a.py

```diff
@@ -0,0 +1,13 @@
+from rcluster import Device
+
+device = Device('device-a', ('224.0.0.64', 22464))
+
+@device.on('device-b/hello')
+def hello_1(data):
+    print('first handler for device-b/hello:', data)
+
+@device.on('device-b/hello')
+def hello_2(data):
+    print('second handler for device-b/hello:', data)
+
+device.run()
```

Comment:
Because `Device.run()` has an infinite loop, this program can't do any regular processing at the same time as listening to subscribed messages. Would the future asyncio changes fix this?

Reply:
The future asyncio integration will be fixing this, currently only pub/sub works.

Reply:
The `@device.every(duration)` can be another approach to have 'concurrent' tasks.